### PR TITLE
[ADD] report_custom_filename

### PR DESCRIPTION
**Pull request by hbrunn**
only ir.actions.report.xml up to now, but it should be simple to figure out where to do that for qweb reports

### CODE REVIEW

File: report_custom_filename/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+##############################################################################
+{
+    "name": "Custom report filenames",
+    "summary": "Configure the filename to use when downloading a report",
+    "version": "1.0",
+    "author": "Therp BV",
+    "license": "AGPL-3",
+    "complexity": "normal",
+    "description": """
+This addon allows for custom filenames for reports. Currently, only old-style
```
Comment (pedrobaeza) [inline report_custom_filename/__openerp__.py]: Please put this in a README.rst file for being used by GitHub and Odoo.

**Comment by pedrobaeza** [comment]
Very interesting module. Can this be backported to 7.0 easily?

**Comment by hbrunn** [comment]
not verbatim, because report generation was quite a bit revamped in 8. also you'd have to monkey patch the controller because orm-style inheritance is new in 8

File: report_custom_filename/__openerp__.py (hunk 1 above)
Comment (nhomar) [inline report_custom_filename/__openerp__.py]: +1 for the comment.

**Comment by nhomar** [comment]
Solved the readme stuff. ILGTM.

**Comment by guewen** [comment]
:+1:

File: report_custom_filename/__openerp__.py (hunk 1 above)
Comment (yvaucher) [inline report_custom_filename/__openerp__.py]: BTW Here is the template
https://github.com/OCA/maintainers-tools/blob/master/template/module/README.rst

**Comment by hbrunn** [comment]
@pedrobaeza @nhomar @yvaucher thanks for your input, there's a README.rst now

**Comment by yvaucher** [comment]
:+1: Thanks

**Comment by pedrobaeza** [comment]
:+1:

**Pull request merged**